dart/biomechanics/MarkerFitter.cpp

```diff
@@ -4811,6 +4847,26 @@ ScaleAndFitResult MarkerFitter::scaleAndFit(
               jointWeights,
               jointAxis,
               axisWeights);
+
+          // Add group scales grad
+          Eigen::VectorXs expNegLogPdfGradient
+              = -alphaScale * expNegLogPdf
+                * fitter->mAnthropometrics->getGradientOfLogPDFWrtGroupScales(
+                    skeletonBallJoints);
+          int groupScaleDim = skeletonBallJoints->getGroupScaleDim();
+          assert(groupScaleDim == expLogPdfGradient.size());
+          assert(
+              jac.cols() == skeletonBallJoints->getNumDofs() + groupScaleDim);
+          assert(
+              jac.rows() > (markerVector.size() * 3)
+                               + (jointsForSkeletonBallJoints.size() * 3));
```

Comment:
Why does this assert use `>` instead of `==`?

Reply:
I'm changing it to `== outputDim`. This is a leftover from trying to track down a dumb bug where I was missing a transpose().